go-apps/meep-loc-serv/api/swagger.yaml

```diff
@@ -3015,28 +3015,6 @@ components:
       properties:
         problemDetails:

```

Comment:
_Details_ section should be replaced with _Notes_ section from MEC Sandbox API files.

Reply:
part of header fix